File: foundation/foundation 2 month/lesson8/homework/ex_12.c
#include <stdio.h>
void check(int k)
{

	for(int i=2;i<k;i++)
	{
		int cnt=0;

		for(int j=1;j<=i;j++)
		{
			if(i%j==0)
			{
				cnt++;
			}
		}
		if(cnt<=2)
		printf("%d ",i);
	}
}


int main()
{
	int n;
	printf("Kiriting: ");
	scanf("%d",&n);
	check(n);
}

File: foundation/foundation 3 month/lesson4/homework/ex_12.py
k=int(input("K ni kiriting: "))
arr=list()
for x in range(1,k+1):
	ms=input(f"{x} chi elementni kiriting: ")
	arr.append(ms)
cnt=0
for x in arr:
	if len(x)>2 and x[0]==x[-1]:
		cnt+=1
print(f"Count: {cnt}")
